src/index.html

```diff
@@ -7,8 +7,37 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
+  <body class="body">
+    <header class="header">
+      <img
+      class="logo"
+      src="images/logo.png"
+      alt="moyo"
+      herf="/src"
```

Comment:
The 'herf' attribute is not valid. If you wanted to make this image a link, you should use 'href' attribute and wrap the image in an 'a' tag.